blueocean-web/src/main/js/DevelopmentFooter.jsx

```diff
@@ -5,14 +5,6 @@ import revisionInfo from '../../../target/classes/io/jenkins/blueocean/revisionI
 
 export class DevelopmentFooter extends Component {
     render() {
-        if (!revisionInfo || !revisionInfo.name) {
-            var blueOceanVersion = document.getElementsByTagName('head')[0].getAttribute('data-blue-ocean-version');
```

Comment:
It's generated at build time but I've seen it missing before... there must have been some reason for the check in the first place.